integration-tests/tests/tests_logfmt.bal

```diff
@@ -317,7 +317,12 @@ public function testObservabilityLogfmt() returns error? {
     io:ReadableCharacterChannel sc2 = new (readableOutResult, UTF_8);
     string outText2 = check sc2.read(100000);
     string[] ioLines = re`\n`.split(outText2.trim());
-    string spanContext = ioLines[1];
+    string spanContext;
+    if (boolean:fromString(getEnvVal("CENTRAL_VERBOSE_ENABLED")) == true) {
+        spanContext = ioLines[ioLines.length() - 1]; // last line  
+    } else {
+        spanContext = ioLines[1];
+    }
```

Comment:
what if we read the last line regardless of whether we enabled verbose or not

Reply:
Will check and update

Reply:
Checked and that approach is working. Hence changed accordingly in 4128bdc